tests/launch-fastify.test.js

```diff
@@ -139,12 +140,55 @@ test('Test fail Fastify creation for invalid options', assert => {
 })
 
 test('Log level inheriting system', async assert => {
+  const stream = new PassThrough()
   let fastifyInstance = await launch('./tests/modules/module-with-log', {})
   assert.strictSame(fastifyInstance.log.level, launch.importModule('./tests/modules/module-with-log').options.logLevel)
   await fastifyInstance.close()
 
-  fastifyInstance = await launch('./tests/modules/correct-module', {})
+  // intercept the stream to check if something is written on it
+  fastifyInstance = await launch('./tests/modules/correct-module', {
+    stream,
+  })
   assert.strictSame(fastifyInstance.log.level, 'info')
+  const test = stream.read()
+  assert.ok(test)
+  await fastifyInstance.close()
+  assert.end()
+})
+
+test('Test custom serializers', async assert => {
+  const stream = new PassThrough()
+  // intercept the stream to check if something is written on it
+  const fastifyInstance = await launch('./tests/modules/correct-module', {
+    stream,
+  })
+
+  stream.read()
+  await fastifyInstance.inject({
+    method: 'GET',
+    url: '/',
+  })
+
+  const logs = stream.read().toString()
+  logs.split('\n').forEach(line => {
+    if (!line) { return }
+    const json = JSON.parse(line)
+    if (json.res) {
+      assert.strictSame(json.res, {
+        statusCode: 200,
+        responseSize: '13',
+      })
+    } else if (json.req) {
+      assert.strictSame(json.req, {
+        requestMethod: 'GET',
+        requestURL: '/',
+        userAgent: 'lightMyRequest',
+        hostname: 'localhost:80',
+        remoteAddress: '127.0.0.1',
+      })
+    }
+  })
+
```

Comment:
this can be written better: see https://github.com/fastify/fastify/blob/master/test/logger.test.js#L33

Reply:
I will look into it.